src/Form/EasyInput.js

```diff
@@ -25,7 +25,7 @@ function EasyInput({ name, Component, ...props }) {
     <PureInput
       name={name}
       value={value !== undefined ? value : defaultValue}
-      error={state.errors[name]}
+      error={renderError ? state.errors[name] : undefined}
```

Comment:
Seems like we've generally established a pattern that `render*` should be a render function. Maybe prefix with should?